charts/observability-pipelines-worker/values.yaml

```
@@ -24,19 +24,28 @@ datadog:
   # datadog.configKeyExistingSecret -- Specify a preexisting Secret that has your configuration key instead of
   # creating a new one. The value must be stored under the `config-key`.
   configKeyExistingSecret: ""
+  # datadog.pipelineID -- Specify your Datadog Observability Pipelines pipeline ID
+  pipelineID:  # <DD_OP_PIPELINE_ID>
+  # datadog.pipelineIDExistingSecret -- Specify a preexisting Secret that has your pipeline ID instead of
+  # creating a new one. The value must be stored under the `pipeline-id`.
+  pipelineIDExistingSecret: ""
   ## Set to 'datadoghq.com' to send data to the US1 site.
   ## Set to 'datadoghq.eu' to send data to the EU site.
   ## Set to 'us3.datadoghq.com' to send data to the US3 site.
   ## Set to 'us5.datadoghq.com' to send data to the US5 site.
   ## Set to 'ddog-gov.com' to send data to the US1-FED site.
   # datadog.site -- The [site](https://docs.datadoghq.com/getting_started/site/) of the Datadog intake to send data to.
   site: datadoghq.com
+  # datadog.ddURL -- The full [URL](https://docs.datadoghq.com/getting_started/site/) of the Datadog intake to send data to (e.g. https://api.datadoghq.com). This overrides the site setting.
+  ddURL: ""
```

```suggestion
  url: ""
```

I think this name looks better especially when written with the parent: `datadog.ddURL` vs `datadog.url`.

---

Yep, sounds good, updated in [cc13d58](https://github.com/DataDog/helm-charts/pull/952/commits/cc13d58f845d5c24df6b474fbad2d4925203a928)